stubs/urllib3/urllib3/util/retry.pyi

```diff
@@ -12,8 +12,15 @@ ResponseError = exceptions.ResponseError
 log: Any
 
 class Retry:
+    DEFAULT_ALLOWED_METHODS: frozenset[str]
```

Comment:
```suggestion
    DEFAULT_ALLOWED_METHODS: ClassVar[frozenset[str]]
```

Since these are class-level constants, it might be good to have them all as `ClassVar`s (should apply similar changes to the other constants as well, if so)